package.json

```diff
@@ -42,29 +42,29 @@
     "@bugsnag/js": "^7.17.0",
     "@bugsnag/plugin-react": "^7.17.0",
     "@floating-ui/react": "^0.19.2",
-    "@opentripplanner/base-map": "^3.0.16",
-    "@opentripplanner/core-utils": "^11.2.3",
-    "@opentripplanner/endpoints-overlay": "^2.0.11",
-    "@opentripplanner/from-to-location-picker": "^2.1.11",
-    "@opentripplanner/geocoder": "^2.0.1",
+    "@opentripplanner/base-map": "^3.0.17",
+    "@opentripplanner/core-utils": "^11.3.0",
+    "@opentripplanner/endpoints-overlay": "^2.0.12",
+    "@opentripplanner/from-to-location-picker": "^2.1.12",
+    "@opentripplanner/geocoder": "^2.0.2",
```

Comment:
Geocoder 2.1.0 will be available shortly

Reply:
will update when I update vehicle rental overlay

Reply:
have assigned you the blocker for consistency

Reply:
Updated!